lib/rules/jsx-sort-props.js

```diff
@@ -44,6 +57,17 @@ module.exports = {
         // Whether alphabetical sorting should be enforced
         noSortAlphabetically: {
           type: 'boolean'
+        },
+        reservedFirst: {
```

Comment:
can we ensure that `additionalProperties` are disallowed on the object form? (not sure if line 73 covers this or not)